include/swift/Sema/ConstraintSystem.h

```diff
@@ -3759,14 +3759,14 @@ class ConstraintSystem {
   Type openUnboundGenericType(GenericTypeDecl *decl, Type parentTy,
                               ConstraintLocatorBuilder locator);
 
-  /// "Open" the given type by replacing any occurrences of unbound
-  /// generic types with bound generic types with fresh type variables as
-  /// generic arguments.
+  /// Replace placeholder types with fresh type variables, and unbound generic
+  /// types with bound generic types whose generic args are fresh type
+  /// variables.
   ///
-  /// \param type The type to open.
+  /// \param type The type on which to perform the conversion.
   ///
-  /// \returns The opened type.
-  Type openUnboundGenericTypes(Type type, ConstraintLocatorBuilder locator);
+  /// \returns The converted type.
+  Type convertInferableTypes(Type type, ConstraintLocatorBuilder locator);
```

Comment:
Subjective nit: I don't love the term "convert/conversion" here, because a type conversion means something different. I would rather use something like `replaceInferableTypes` or `substituteInferableTypes`. We could also go for the long form and tack on `WithTypeVar[iable]s` at the end depending on how verbose we want to be 🙂

Reply:
That makes sense to me re: "convert"! I'll swap this out with one of your suggestions.